test/robot/report/report.proto

```diff
@@ -29,8 +30,21 @@ message Input{
   string gapit = 2;
   // Gapis is the stash id of the graphics analysis server to use.
   string gapis = 3;
+  // GapidApk is the stash id of the gapid.apk to use for a Vulkan trace.
+  string gapid_apk = 4;
+  // Layout represents the ABI details for Vulkan report.
```

Comment:
ToolingLayout

Reply:
Changed `layout` to `tooling_layout`, so the generated code will be `ToolingLayout`